packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTabs/CWTabsRow.scss

```diff
@@ -3,4 +3,5 @@
 .CWTabsRow {
   display: inline-flex;
   border-bottom: 1px solid $neutral-200;
+  margin-bottom: 8px;
```

Comment:
can we give it a fullWidth prop?

Reply:
we should not do that. Take a look at my comment to see how this should work https://github.com/hicommonwealth/commonwealth/pull/5367#issuecomment-1772644150